contents/handbook/engineering/feature-pricing.md

```diff
@@ -6,38 +6,61 @@ showTitle: true
 
 ## In an ideal world, Posthog’s pricing enables users and organizations to:
 
-- Use PostHog for free if they are hobbyists or pre-PMF
-- Experience the product before paying for it
-- Start paying when they are ready, on their own, with few hurdles
-- Transparently pay for the value they receive
-  - Eg. Usage-based pricing on events, recordings
-  - Eg. Paying per product so they only pay for what they use
-- Make it a no-brainer to pick PostHog over other competitors
+1. Use PostHog for free if they are hobbyists or pre-PMF.
+2. Experience the product before paying for it.
+3. Start paying when they are ready, on their own, with few hurdles.
+4. Transparently pay for the value they receive.
+  - e.g. Usage-based pricing on events, recordings.
+  - e.g. Paying per product, so they only pay for what they use.
+5. Make it a no-brainer to pick PostHog over other competitors.
 
 
 ## Our goals with these principles are to: 
 
-- Keep the engineers at PostHog as close to our customers as possible so that they can build new product or improve existing in ways that are most impactful for them
-- Maintain low barriers to entry for our customers so they can see value in PostHog quickly
-- Ensure transparency around the value we provide to our customers
-- Tightly couple our success with that of our customers’ - the more we can help them succeed, the more we will succeed
-  - Eg. with usage-based pricing
+- Keep the engineers at PostHog as close to our customers as possible, so they can build new products or improve existing in ways that are most impactful for them.
 
-It's important that every time we create a new feature or shift our pricing plans we evaluate the changes alongside these principles to make sure we are making changes that align with our pricing values.
+- Maintain low barriers to entry for our customers, so they can see value in PostHog quickly.
+
```

Comment:
Will these extra line breaks break up the list?

Reply:
Erm, maybe? I don't feel super strongly about this particular example, but I prefer some line breaks when the list items are quite long, but this is a bit of mixture. Will revert.